docs/source/temp_sense_genCollab.ipynb

```diff
@@ -0,0 +1,10 @@
OPRo888ki5pmo6jqvf6NGjXe7VakKM4zXXXOMIjJoR1m+33XarqQluODDgNiNYu2Ch/Na3vlW1mRdffDG6++676yIWymt0+PDhNd0Y//Wvf0VXXHFFRLqdZoRct6S/4DdSTWDhJE9vM0I/u+yySyplULn2WL+bbrqp4d8i1xtrcvDBB7t1lAgBISAEhIAQEAJCoJMRIFPEf/7zn4aHiD6w+OKL59bnXRxdAd0D4tEiUp65jmBZl/QuApbDExb9Qj477LBDTUCNGTOmkP4s1iz++OOPq/ZpCkch/YHTueeeW7U/i+OLjfG1kD5t06FqfxSYNGlSIf0xx6eeeqpqn3Pnzi2sP8Zei1hKoUL6ZG1Yo2rCWhf127ANqGrduWuZa7qIPseOHVu1Pwpsv/32hfRnO7819adCQkAICAEh0D8IWHhJbK66/TPhLp6p8VLEc+bM6eIZ1D50S7XY1LvPUUcdldvZeeedF5sxzbVtKR3jN954I7dcIwdlIWdXoodl8ODB0U477RQ9++yzTc0SqxzuGbUI/f3hD39o2Crn+1hnnXWihRdeuGqXpHIbNmxY0xZyCIRGjhxZtT8s21gLsXY2I7RTq7UaS/p9993XtIWc62HFFVesOmzKbLbZZoVYyBl7LQIWWK3tRlZL8bJl8KrIIyTKVmCt8cKoJSdxtm74Hes/11814VrmN/TMM89UK1rxPL9FfmO1COVIn+RTEdVSJ1uG/vbZZ5/sYX0XAkJACAiBPkbg7LPPjk4//XT3PlTrc76P4Wrb1D379/nnnx+dfPLJLmd2L8vbb78dvfvuu01NccMNNyypT9aPww8/3L2jrrLKKu6d3BT/knKNHlAe8kaRUz0hIASEgBAQAkJACAgBIdCHCJATnhhaQpouuuiiPkSgO6Zs3nnRYost5gZ71llnRT/5yU+6Y+ANjhJXdVzW82Tq1KkRGxPI5ZdfXtYIh6I933zzpZrARR3DEe1feeWV0aKLLprXRcPHZCFvGDpVFAJCQAgIASEgBISAEBACQkAICIFOQOCrX/1qNGTIkNyhWBhvcny77bYrGyeeV5lUkfBjtUrEst4qZNWuEBACQkAICAEhIASEgBAQAkJACLQVAVzNn3zySTeGFVZYoS5lfCAGLoV8IFBWH0JACAgBISAEhIAQEAJCoMsROPTQQ6Pvf//7CafNvffe677zycbuogTNnDkz2nnnnaNBgwZF888/v1OEiGP++c9/7nhj8sTI4pI2+f8nn3wSHXfccS52FwbsJZdc0mXhIY7XC2WOOeaYaOjQoa4fXIq32WabVJmwrx//+Meuj1tvvTX673//69y511prrQg+oSWWWCIaMWJENH369JrYuu+55x7nzkwmG9/3Rhtt5Nr8+9//njfFaMqUKa7/iy++2Ln+H3jggS6FFnMbP3589NJLL6XqPfzww9FBBx3kuG/griGLDVlevvOd70SHHXZYFFp/fUXi+/fff/+kneuuuy7BlTkjjJFx/OhHP8odJweNKDepR8amUOCo8usPX82NN97o1oDc3N/97nejq666KlWea+KWW25xXDhLL7106prApRwsWiGvvvpqwjW1wQYbtKKL5tpshAlOdYSAEBACQkAICAEhIASEgBDoLwRWXnnlsgzWpkQmYFgq2njUqFFly5r2EpsCG5tCVwKgKU9JPVPeYlNSc9shc4kpfLGlTY1NucstAys2ZbKy0korufJHHHFEbEptUtcUyVRGFCMYjm2jIVvdff/ss8/i733ve7n90jYf24CILVVuSX1TSN15I02NyWLky/u/DzzwgKtjmxaxpTotOZ8tbzHP8W233Zbqx+L8y9YzQltX1ly3XRmzGpeM0R+w+OmkHVP+U+UsNWpyjowz2WwytrmQlLcUvrERBpcdE+OAvdzSw5YdS6MnLD1y0u+FF17YaDMtqycLua2+RAgIASEgBISAEBACQkAICIHKCGDdtNSoCemVKYvuOx9TplxlrK8wr99///0u68kPfvCD6IknnnDWYgi3LrnkEmflfeuttyJT2iOYscsJWT5MuY+OPfbY6He/+11kylpkaUudZRXLLdZh4oGxkJ966qnR448/Hv3+97+PzjjjDGdFNg0qMqXbjSlPsMpaOtiIeVgaq+jTTz+N/vrXv0ZnnnlmRNwwmVGwxkPqFQrWYKy8pgS7w3vttVdkirer/+GHH0amADoWbv6/9dZbl1i8fVum0Ea33357tPzyy0dY7WkHqzeZbpAjjzwyiV2eMGGCw+Cjjz6KTLl1mGM1NyXYzQ8rO6zqXh555JGUh4CloU3WCvyKFgj+wJxxMhes+d5CDxkaHguzZ89218QPf/hDd03YpobzrIAYEIv/m2++6a6Jd955p9DhcV14yWNRL7SzRhprmaqvhoWAEBACQkAICAEhIASEgBDoOQRMoXMWR1PCSuZmymxVa+Qrr7wSewuupQRNtRFayOnjrrvuKunDFPSkD1MCY1NOS8ocf/zxSZnHHnssdd5byGnf3M1jU+5L6k+bNi2pb+zcqfO/+MUvknOnnHJKSV0O2EZDbG7ortwmm2ySKuMt5JzbeOONnSXcix+LKd6xbQq4+lji88ZIHVPEk7E8/fTTqX5og/p8zD09dY4vRVrIv/KVr8Sm+Kb68GO2zZJkHJdeemnJODhgGymxv67GjRuXW6bRgxaO4Pqn/c8//7zRZlpWTxZyWx2JEBACQkAICAEhIASEgBAQAs0hYBpLdMEFF7hG1ltvPWcJzRNzfXcp0xAsxOUsolgzsYBnBSuqF3P5dlblrJgSnBzKxrf7E7Bymwuzs9pmBevu6quv7g5nY6F9+iwIwrDe5wlx81i4kYceeih64YUX8opFtnHgLP5e/FjwKCD3NTHaxMfnjZE6m2++eVIXC327ZOzYsS5uPBTGjCcDGCOcx5KfJ6uuuqqz+CN4YsAfUIT885//dF4TyLBhwyLbOCii2ULbkEJeKJxqTAgIASEgBISAEBACQkAI9CcCkJ
```

Comment:
1. Add a separate cell in the beginning of the notebook as a header for the sphinx to include it in docs (Don't edit here, you might miss the syntax. Edit with the notebook running in the browser.) Something like below - 
![image](https://user-images.githubusercontent.com/100982953/190466536-21629f11-e395-404d-b7d8-3dfc54e51ced.png)
2. Also run all the cells and add that copy of notebook to the docs. Seeing the result of each cell will be really good in the docs.